README.md

```diff
@@ -8,6 +8,15 @@ Join the [conversation on Slack](https://tessel-slack.herokuapp.com/), our proje
 ## Installation
 Clone this repo and then run `npm link`.
 
+## Updating Tessel 2 On-board OS/Firmware
+There are two components to the Tessel on-board software: The OpenWRT Linux image on the MediaTek processor and the firmware image that runs on the Atmel co-processor. 
+
+### How do I know if I need to update my T2? 
+Unfortunately, [we have an open PR](https://github.com/tessel/t2-cli/pull/130) to detect the version of code running on T2 but it hasn't merged yet! If you aren't seeing your Tessel 2 show up with `t2 list`, then you probably have an old version of firmware (because the USB VID/PID is out of date). If you see functionality in the [Tessel API](https://github.com/tessel/t2-firmware#t2-hardware-api) that isn't defined on your board, then you probably need to update.
+
+### Updating
+Eventually, this will be [rolled into the CLI](https://github.com/tessel/t2-cli/issues/81) but it hasn't been built yet. For now, go ahead and update your co-processor firmware by following the instructions [here](https://github.com/tessel/t2-firmware/#flashing). Then update the OpenWRT image on the primary processor by either [building it yourself](https://github.com/tessel/openwrt-tessel) or downloading [a prebuilt binary](https://tessel.slack.com/archives/engineering/p1435040051000036), transferring it to your T2 (often accomplished with `scp -i ~/.tessel/id_rsa PATH_TO_IMAGE root@IP_ADDR_OF_T2:/tmp`) and running `sysupgrade /tmp/PATH_TO_IMAGE` on your T2. Make sure it's uploaded to the `/tmp` folder on T2 - any other folder will ruin the upgrade. If you need to find the IP Address of your T2 but don't know how, first connect it to your network (`t2 wifi -n YOUR_SSID -p YOUR_PASSWORD`), then run `t2 list` to make sure it's actually connected over the LAN, and note its name. Then run `ping YOUR_NAME.local` in order to see the IP address (you can also give it a simpler name first with `t2 rename NEW_NAME`. If you already know what dterm is, you can just dterm into the device and run `ifconfig`. Wait for it to automatically restart itself and you're good to go!
```

Comment:
The `#flashing` url fragment isn't present here: https://github.com/tessel/t2-firmware/#flashing